controllers/college-controller.js

```diff
@@ -0,0 +1,36 @@
+import { collegeModel } from "../models/college-model.js";
+
+
+export const getCollege = async (req, res, next) => {
+    try {
+        const findCollege = await collegeModel.find()
+        res.json(findCollege)
+    } catch (error) {
+        next(error)
+        
+    }
+}
+
+
+export const postCollege = async (req, res, next) => {
+    try {
+        const addCollege = await collegeModel.create(
+            req.body
+        )
+res.json(addCollege)
```

Comment:
check your indentation on line 20

you can right click and click on format document for that